pytorch_pfn_extras/handler.py

```diff
@@ -27,6 +27,9 @@ def torch_autocast(enabled=True):
         yield
 
 
+_is_torch_17 = torch.__version__.split('.')[1] == '7'
```

Comment:
This is horrible, minimum supported is 1.7 so I just check against it like this ...